web-client/src/providers/realUserMonitoring.ts

```diff
@@ -0,0 +1,30 @@
+import { AwsRum, AwsRumConfig } from 'aws-rum-web';
+
+export const initializeRealUserMonitoring = () => {
+  if (process.env.ENV === 'local') return;
+  let awsRum: AwsRum;
+  try {
+    const config: AwsRumConfig = {
+      allowCookies: true,
+      enableXRay: false,
+      endpoint: 'https://dataplane.rum.us-east-1.amazonaws.com',
+      identityPoolId: process.env.RUM_IDENTITY_POOL_ID,
+      sessionSampleRate: process.env.ENV === 'prod' ? 0.1 : 1, // sample 10% of sessions in production until we know how expensive this will be, 100% in lower environments
+      telemetries: ['performance'],
```

Comment:
There are other potentially useful metrics to collect, but starting with "performance" since this story is solely focused on ensuring we can measure performance. We also want to gather more data about how expensive this will be to run in production.